src/azure-cli-core/azure/cli/core/__init__.py

```diff
@@ -370,6 +371,8 @@ def load_arguments(self, command=None):
                 with loader.argument_context('') as c:
                     c.argument('resource_group_name', resource_group_name_type)
                     c.argument('location', get_location_type(self.cli_ctx))
+                    c.argument('vnet_name', vnet_name_type)
+                    c.argument('subnet', subnet_name_type)
```

Comment:
This is somehow contradicting with my understanding about `vnet_name` and `subnet`. But, yea, they can only be configured as global arguments.